src/index.ts

```diff
@@ -2,9 +2,11 @@ import { Response, Request, Headers as FetchHeaders, RequestInfo, RequestInit }
 import { AxiosInstance, AxiosRequestConfig } from './types';
 import FormData from 'form-data';
 
-export type AxiosTransformer = (config: AxiosRequestConfig, input: RequestInfo, init?: RequestInit) => AxiosRequestConfig;
+export type FetchInit = RequestInit & { extra?: any }
```

Comment:
The idea was that any extra options (a superset of RequestInit) would be passed to the transformer, not just `extra`

Reply:
It's be fine if you did some hacky typing in the test code to make it work